src/unix/linux-core.c

```diff
@@ -552,6 +552,29 @@ int uv_uptime(double* uptime) {
 }
 
 
+int uv_cpu_num() {
```

Comment:
make it static and use `uv__` as the prefix, we use that for internal libuv functions.